src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserType {
```

Comment:
```suggestion
export interface User {
```

Reply:
In code quite easier to understand, that it is type or interface by usage, no need to name with type word